### DISCUSSION TITLE
Atoti 0.7.4 Release: Problems with Docker Image

### DISCUSSION THREAD

**Discussion by brunoguic** [atoti/atoti#775] [Q&A] [answered]
Hello everyone!

I am currently using Atoti Docker and recently, on March 15th, Atoti 0.7.4 was released. Since then, there has been a 14-day countdown to migrate to the latest version. However, the Atoti team has not released a new image on Docker Hub yet, causing some issues for users.

Has anyone else faced this problem? And does anyone know how we can get in touch with the Atoti team for support?

**Comment by tibdex** [comment, marked as answer]
Hi Bruno, you're right that https://hub.docker.com/r/atoti/atoti hasn't been updated for 0.7.4. Can I ask you what's your use case for this Docker image?

It's just `jupyter/base-notebook` with `atoti[jupyterlab]` and `jupyter-server-proxy` installed:

```dockerfile
FROM jupyter/base-notebook:lab-3.6.1

# Add Conda channel like base-notebook does it.
RUN conda config --system --prepend channels https://conda.atoti.io

RUN mamba install --yes atoti-jupyterlab=0.7.4 jupyter-server-proxy
```

If you rely on this Docker image how do you install your other dependencies once you have a Docker container running?

**Reply by brunoguic** [reply]
Hello Thibault. Thanks for your orientation.

My use case is simply running the notebooks to analyze some data from my research. Don't have any other dependency.

I didn't know how simple is to set up from jupyter/base-notebook. I will follow your Dokerfile.

Thanks
